script/QA/QA_02.txt

```diff
@@ -22,7 +22,7 @@ If you make a choice while naively thinking,%{n}
 -- Page 32, Offset 42825.
 -- 消極的なメガネくんに喝！　を入れる救済コーナー。
 // Rather than half-hearted, Shiki is being reprimanded here for his passivity. Whether that comes across well with a word like half-hearted I'm not sure.
-Ambivalent Glasses-kun STRIKE! And thus you enter this%{n}rescue corner.
+Ambivalent four-eyes STRIKE! And thus you enter this%{n}rescue corner.
```

Comment:
Four-Eyes maybe

Reply:
i think Ben and Ross preferred not to have nicknames capitalized

Reply:
correct